social_panel/lib/twitter_client.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'faraday'
+require 'JSON'
+require 'pry'
+
+class TwitterClient
+  URL = 'https://api.twitter.com/1.1'
+  ACCESS_TOKEN = "Bearer #{ENV['SOCIAL_PANEL_TWITTER_ACCESS_TOKEN']}"
+
+  # def initialize(access_token: ENV['SOCIAL_PANEL_TWITTER_ACCESS_TOKEN'])
+  #   @access_token = "Bearer #{access_token}"
+  # end
+
+  def get_tweet_by_id(tweet_id)
+    response = Faraday.get("#{URL}/statuses/show.json?id=#{tweet_id}") do |request|
+      request.headers['Authorization'] = ACCESS_TOKEN
+    end
```

Comment:
As a next step of your refactor: try to use this syntax:

```ruby
url = 'http://sushi.com/nigiri/sake.json'
resp = Faraday.get(url, {a: 1}, {'Accept' => 'application/json'})
# => GET http://sushi.com/nigiri/sake.json?a=1
```
I copied it from the doc: https://lostisland.github.io/faraday/usage/

Note that you will be refactoring here internal things in this class. You can rely on your tests. The tests should still pass.